docs/how_to/custom_plugins.rst

```diff
@@ -361,6 +361,11 @@ new plugin::
         title = models.CharField(max_length=50)
 
         def copy_relations(self, oldinstance):
+            # Before coping related objects from old instance current one need
+            # to be deleted or at public version of the page can appear
+            # duplicate items.
+            self.associated_item.all().delete()
```

Comment:
"Before copying related objects from the old instance, the ones on the current on need to be deleted. Otherwise, duplicates may appear on the public version of the page."